core/config/engine.cpp

```diff
@@ -294,6 +295,18 @@ void Engine::get_singletons(List<Singleton> *p_singletons) {
 	}
 }
 
+void Engine::add_extension_class_doc(int p_compressed_size, int p_uncompressed_size, PackedByteArray p_compressed_data) {
+	ExtensionClassDoc doc;
+	doc.compressed_size = p_compressed_size;
```

Comment:
Is it necessary? `PackedByteArray` already have `size()` method, so only `uncompressed_size` is useful. Using array `size()` is also safer and ensure it won't read outside the array.

Reply:
Removed :)